src/icons/fileIcons.ts

```diff
@@ -1461,6 +1461,7 @@ export const fileIcons: FileIcons = {
     { name: 'chess', fileExtensions: ['pgn', 'fen'], light: true },
     { name: 'gemini', fileExtensions: ['gmi', 'gemini'] },
     { name: 'sentry', fileNames: ['.sentryclirc'] },
+    { name: 'js-beautifier', fileNames: ['.jsbeautifyrc', '.jsbeautify.json'] },
```

Comment:
Looks good, but `.jsbeautifyrc` is already registered as a file name for JSON icons, see line 30. Could you please remove the association from the JSON icon?


```diff
{
  name: 'json',
  fileExtensions: ['json', 'tsbuildinfo', 'json5', 'jsonl', 'ndjson'],
  fileNames: [
    '.jscsrc',
    '.jshintrc',
    'composer.lock',
-    '.jsbeautifyrc',
    '.esformatter',
    'cdp.pid',
    '.mjmlconfig',
    '.lintstagedrc',
  ],
},
```